benchmarks/dungeon-benchmark/build.gradle.kts

```diff
@@ -13,12 +15,12 @@ android {
   compileSdk = libsCatalog.version("compileSdk").toInt()
 
   compileOptions {
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
+    sourceCompatibility = javaTargetVersion
```

Comment:
Do we really need to change `sourceCompatilibity` too? Will that, say, prevent us using language features that aren't actually supported?

Reply:
That's correct keeping them in sync will prevent us from inadvertently using Java language features that don't exist in the target environment.